src/transformers/onnx/features.py

```diff
@@ -223,6 +224,15 @@ class FeaturesManager:
             "token-classification",
             onnx_config_cls=GPT2OnnxConfig,
         ),
+        "gpt-j": supported_features_mapping(
+            "default",
+            "default-with-past",
+            "causal-lm",
+            "causal-lm-with-past",
```

Comment:
I think you can add the `question-answering` feature since GPT-J has a question answering head (see the `GPTJForXxx` classes in `modeling_gptj.py`)
```suggestion
            "causal-lm-with-past",
            "question-answering",
```